diff --git a/src/lib/libraries/extensions/index.jsx b/src/lib/libraries/extensions/index.jsx
index ba18b91..4d578e3 100644
--- a/src/lib/libraries/extensions/index.jsx
+++ b/src/lib/libraries/extensions/index.jsx
@@ -47,6 +47,15 @@ import gdxforConnectionIconURL from './gdxfor/gdxfor-illustration.svg';
 import gdxforConnectionSmallIconURL from './gdxfor/gdxfor-small.svg';
 
 export default [
+    {
+        name: 'Your Extension',
+        extensionId: 'yourScratchExtension',
+        iconURL: musicIconURL,
+        insetIconURL: musicInsetIconURL,
+        description: 'Your custom Scratch extension',
+        featured: true,
+        disabled: false
+    },
     {
         name: (
             <FormattedMessage
